src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -136,6 +137,27 @@ def _read_metadata_file(meta_filename: str) -> dict:
     return meta_info
 
 
+def _get_genome_obj_meta(
+        genome_info: list[Any],
+) -> dict[str, Any]:
+    # retrieves workspace genome object metadata and convert it to a dict with parsed values
+
+    kb_genome_meta = genome_info[-1]
```

Comment:
Probably unlikely, but we could possibly add more items to the `object_info` tuple
```suggestion
    kb_genome_meta = genome_info[10]
```

Reply:
👍